plugins/lando-core/scripts/user-perm-helpers.sh

```diff
@@ -49,17 +49,17 @@ reset_user() {
   if getent group "$HOST_GID" 1>/dev/null 2>/dev/null; then
     HOST_GROUP=$(getent group "$HOST_GID" | cut -d: -f1)
   fi
-  if [ "$DISTRO" = "alpine" ]; then
+  if command -v usermod > /dev/null 2>&1 && command -v groupmod > /dev/null 2>&1; then
+    usermod -o -u "$HOST_UID" "$USER" 2>/dev/null
+    groupmod -g "$HOST_GID" "$GROUP" 2>/dev/null || true
+    usermod -g "$HOST_GID" "$USER" 2>/dev/null || true
+    usermod -a -G "$GROUP" "$USER" 2>/dev/null || true
+  else
```

Comment:
I don't know what exactly has changed in Composer v2, but it looks like due to some race conditions, `pre-start` commands may run right between the runs of `deluser` and `adduser` :-)

Therefore, it is wise to use `usermod` and `groupmod` regardless of the distro, and fall back to `adduser`/`deluser` if `usermod` or `groupmod` is unavailable.